toonz/sources/toonz/xshcellviewer.cpp

```diff
@@ -3344,6 +3344,9 @@ void CellArea::createCellMenu(QMenu &menu, bool isCellSelected, TXshCell cell) {
     menu.addAction(cmdManager->getAction(MI_Cut));
     menu.addAction(cmdManager->getAction(MI_Copy));
     menu.addAction(cmdManager->getAction(MI_Paste));
+    if (!soundTextCellsSelected) {
+      menu.addAction(cmdManager->getAction(MI_PasteDuplicate));
+    }
```

Comment:
In our studio, basically they don't (shouldn't) change the drawings in the levels during compositing workflow. I'm bit concerning that users may use this command by mistake while working on Xsheet.

I'm not sure how often you are estimating this command will be used, but how about moving this command into the `Paste Special` submenu?

Reply:
I personally would use Paste as Duplicate pretty much every time over regular Paste, but if it’s in a right click submenu rather than the main right click menu (and since it has a configurable keyboard shortcut) that would still be okay for me, I see what you’re saying about this causing issues in some workflows!

Reply:
I moved it to the paste special menu.